### ISSUE TITLE
Regarding Deprecated QR Code API

### ISSUE THREAD

**Issue by myetcm** [google/google-visualization-issues#3000]
Hello Google Chart Support Team,

We've reviewed the documentation and are aware that the QR Code API is deprecated. However, we have been using it for the past five years. Could you please confirm whether the QR Code (https://chart.googleapis.com/chart?chs=150x150&cht=qr&chl=Hello%2C+World%21) is currently not supported or working?

Thank you for your assistance.

**Comment by orwant** [comment]
It is intentionally down for several hours every day. We've started rolling
blackouts so that the last few remaining users can learn that they're
depending on a 12-year deprecated service.

**Comment by i2staies** [comment]
yes,i have the same question , so what can i do for selve it

**Comment by chrisb123** [comment]
That's fine, but what is the replacement?

**Comment by orwant** [comment]
You can find suggestions in the archive of this list.

**Comment by chrisb123** [comment]
Sorry, I cant seem to find the list you are referring to.

**Comment by herichi** [comment]
any solution update on this?

**Comment by chrisb123** [comment]
No, every things always points to the depreciate api. How are we supposed to know its depreciate if all documents on all the google support page mentioned it say to use the depreciated API.
How we supposed to use the replacement API if it is impossible to find?
